src/library/searchqueryparser.cpp

```diff
@@ -109,6 +110,16 @@ QString SearchQueryParser::getTextArgument(QString argument,
 void SearchQueryParser::parseTokens(QStringList tokens,
                                     QStringList searchColumns,
                                     AndNode* pQuery) const {
+    // we need to create a filtered columns list that are handled differently
+    QStringList(queryColumns);
```

Comment:
This is a **very** uncommon style to declare a local variable! I wasn't even aware that the compiler accepts this construct that looks like invoking a single argument constructor of QStringList and instantly dropping the result.
```
QStringList queryColumns;
```